src/lib.rs

```diff
@@ -243,13 +247,26 @@ unsafe impl Zeroable for Price {}
 unsafe impl Pod for Price {}
 
 impl Price {
+  /**
+   * Get the current status of the aggregate price.
+   * If this lib is used on-chain it will mark price status as unknown if price has not been updated for a while.
+   */
+  pub fn get_current_status(&self) -> PriceStatus {
```

Comment:
maybe leave a comment on the `agg.status` that users should call `get_current_status()` instead of reading the field directly?

Reply:
Added a comment in PriceInfo